src/Components/ProductCard/ProductCard.jsx

```diff
@@ -0,0 +1,55 @@
+import { useProductList } from "../../Context/index";
+import { BsFillStarFill } from "react-icons/bs";
+import { AiOutlineHeart } from "react-icons/ai";
+
+export const ProductCard = () => {
+  const { productList } = useProductList();
+  return (
+    <>
+      {productList.map((item) => (
```

Comment:
You can destructure the item here, so that you don't need to write item. everywhere

Reply:
okay